contrib/localnet/orchestrator/smoketest/main.go

```diff
@@ -284,7 +284,7 @@ func LocalSmokeTest(_ *cobra.Command, _ []string) {
 	smokeTest.CheckZRC20ReserveAndSupply()
 
 	smokeTest.TestBitcoinWithdraw()
-	smokeTest.CheckZRC20ReserveAndSupply()
+	//smokeTest.CheckZRC20ReserveAndSupply()
```

Comment:
Is there a specific reason commenting this? Does the PR affect `TestBitcoinWithdraw`?